### ISSUE TITLE
Task decorator toggle non-functional

### ISSUE THREAD

**Issue by nizeii** [ivan-lednev/obsidian-day-planner#536]
**Describe the bug**
Task decoration toggle doesn't appear to do anything?

**To Reproduce**
1. Navigate in Settings to the Day Planner menu.
2. Click on toggle switch in Task decoration.
3. Close menu.
4. No apparent UI changes occur in the timeline. It defaults to having the timestamp appear in the task header (e.g. 11:00 - 11:30 Work out).

**Expected behavior**
Time stamps either being removed from timeline tasks or put in place upon toggle.

Also, thanks for all the work you put in!

**Comment by lilskippyy** [comment]
> **Describe the bug** Task decoration toggle doesn't appear to do anything?
> 
> **To Reproduce** Steps to reproduce the behavior:
> 
> 1. Navigate in Settings to the Day Planner menu.
> 2. Click on toggle switch in Task decoration.
> 3. Close menu.
> 4. No apparent UI changes occur in the timeline. It defaults to having the timestamp appear in the task header (e.g. 11:00 - 11:30 Work out).
> 
> **Expected behavior** Time stamps either being removed from timeline tasks or put in place upon toggle.
> 
> Also, thanks for all the work you put in!

happens the same to me

**Comment by vmenezio** [comment]
I'm having the same issue. Is there a different setting that can be used to disable these timestamps? I tried hiding them with CSS but they don't appear to be in a different element from the rest of the text of the time block.

```html
<ul class="contains-task-list">
  <li data-task="" class="task-list-item" dir="auto">
    <input type="checkbox" class="task-list-item-checkbox" data-line="27">
      19:00 - 21:30 workout
  </li>
</ul>
```

**Comment by rahmanidashti** [comment]
Is this fixed?

**Comment by vmenezio** [comment]
> Is this fixed?

I don't believe so. There is an open pull request that should address this from a few months back (https://github.com/ivan-lednev/obsidian-day-planner/pull/702), but conflicting changes have since been introduced to the code base, so it would need to be updated again in order to be merged.

**Comment by ivan-lednev** [comment]
Please update and re-check.
